x-pack/plugins/apm/public/components/app/service_inventory/service_list/order_service_items.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { orderBy } from 'lodash';
+import { ServiceHealthStatus } from '../../../../../common/service_health_status';
+import {
+  ServiceListItem,
+  ServiceInventoryFieldName,
+} from '../../../../../common/service_inventory';
+
+type SortValueGetter = (item: ServiceListItem) => string | number;
+
+const SERVICE_HEALTH_STATUS_ORDER = [
+  ServiceHealthStatus.unknown,
+  ServiceHealthStatus.healthy,
+  ServiceHealthStatus.warning,
+  ServiceHealthStatus.critical,
+];
+
+const sorts: Record<ServiceInventoryFieldName, SortValueGetter> = {
+  [ServiceInventoryFieldName.HealthStatus]: (item) =>
+    item.healthStatus
+      ? SERVICE_HEALTH_STATUS_ORDER.indexOf(item.healthStatus)
+      : -1,
+  [ServiceInventoryFieldName.ServiceName]: (item) =>
+    item.serviceName.toLowerCase(),
+  [ServiceInventoryFieldName.Environments]: (item) =>
+    item.environments?.join(', ').toLowerCase() ?? '',
+  [ServiceInventoryFieldName.TransactionType]: (item) =>
+    item.transactionType ?? '',
+  [ServiceInventoryFieldName.Latency]: (item) => item.latency ?? 0,
```

Comment:
Is [this](https://github.com/elastic/kibana/blob/398520ba2329a9a64fd99773069c4349fa901ac3/x-pack/plugins/apm/public/components/app/service_inventory/service_list/index.tsx#L363) no longer relevant and the default is now `0`?

Reply:
good point, it's not relevant for throughput I think (because it will always be reported as 0), but for latency it will be.

Reply:
and definitely failure rate